src/lavinmq/replication/server.cr

```diff
@@ -209,21 +221,41 @@ module LavinMQ
 
         def read_acks(socket = @socket) : Nil
           spawn action_loop, name: "Follower#action_loop"
+          @server.wait_for_followers.try_send nil
           loop do
             len = socket.read_bytes(Int64, IO::ByteFormat::LittleEndian)
             @acked_bytes += len
-            @ack.try_send nil
+            if max_lag = Config.instance.max_lag
+              if lag < max_lag
+                while @ack.try_send lag
```

Comment:
```suggestion
                while @ack.try_send lag
```
Will `lag` ever change while in this loop? Doesn't this mean that the same `lag` will be sent until the channel is clogged? Just `@ack.send lag` (without looping) will do the same?

Reply:
.send does not work for some reason.. the publishes do not stop even tough the replication sevrer has to few followers.. but this approach with try_send works as it should, so i'll apply that instead!